src/__tests__/components/ProfileMenu/ProfileMenu.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Adapter from 'enzyme-adapter-react-16';
+import EnzymeToJson from 'enzyme-to-json';
+import { mount, configure } from 'enzyme';
+
+import ProfileMenu from '../../../components/ProfileMenu/ProfileMenu.component';
+
+beforeAll(() => {
+  configure({ adapter: new Adapter() });
+});
+
+describe('ProfileMenu', () => {
+  it('Renders ProfileMenu', () => {
+    const auxFunc = () => true;
+    const profileMenu = mount(
+      <ProfileMenu isProfileMenuActive={null} closeMenu={auxFunc} />
+    );
+    expect(EnzymeToJson(profileMenu)).toMatchSnapshot();
```

Comment:
This could be:
```
const profileMenu = mount(
  <ProfileMenu isProfileMenuActive={null} closeMenu={jest.fn()} />
);
expect(profileMenu).toMatchSnapshot();
```
This way you are using the functionality provided by jest to create mock functions. On the other hand, you don't need to get the `json` representations to make the snapshot assertion, this only represents a burden to the engine.